packages/amplify_core/test/io/aws_file_io_test.dart

```diff
@@ -0,0 +1,78 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@TestOn('vm')
+
+import 'dart:convert';
+import 'dart:io' as io;
+
+import 'package:amplify_core/src/io/aws_file_platform_io.dart';
+import 'package:test/test.dart';
+
+import 'utils.dart';
+
+void main() {
+  group('AWSFile io implementation', () {
+    String currentPath = io.Directory.current.path;
```

Comment:
```suggestion
    final currentPath = io.Directory.current.path;
```

Reply:
`currentPath ` actually might be reassigned in the following lines. Changed to `var currentPath =`